<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWidget</class>
 <widget class="QWidget" name="SettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>300</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupSetting">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>300</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>나눔스퀘어_ac Bold</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="hLayout1">
        <item>
         <widget class="QGroupBox" name="groupAbility">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>150</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>나눔스퀘어 네오 ExtraBold</family>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>특성</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="hLayoutNeck">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>나눔스퀘어 네오 ExtraBold</family>
                 <pointsize>12</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>목걸이</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lbNeck1">
               <property name="font">
                <font>
                 <family>나눔스퀘어 네오 Bold</family>
                 <pointsize>12</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lbNeck2">
               <property name="font">
                <font>
                 <family>나눔스퀘어 네오 Bold</family>
                 <pointsize>12</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="hLayoutEar">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>나눔스퀘어 네오 ExtraBold</family>
                 <pointsize>12</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>귀걸이</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lbEar1">
               <property name="font">
                <font>
                 <family>나눔스퀘어 네오 Bold</family>
                 <pointsize>12</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lbEar2">
               <property name="font">
                <font>
                 <family>나눔스퀘어 네오 Bold</family>
                 <pointsize>12</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="hLayoutRing">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>나눔스퀘어 네오 ExtraBold</family>
                 <pointsize>12</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>반지</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lbRing1">
               <property name="font">
                <font>
                 <family>나눔스퀘어 네오 Bold</family>
                 <pointsize>12</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lbRing2">
               <property name="font">
                <font>
                 <family>나눔스퀘어 네오 Bold</family>
                 <pointsize>12</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gruopSetEffect">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>150</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>나눔스퀘어 네오 ExtraBold</family>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>세트 효과</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="lbSetEffect">
             <property name="font">
              <font>
               <family>나눔스퀘어 네오 Bold</family>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupClassEngrave">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>150</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>나눔스퀘어 네오 ExtraBold</family>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>직업 각인</string>
          </property>
          <layout class="QVBoxLayout" name="vLayoutClassEngrave"/>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupEngrave">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>120</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>나눔스퀘어 네오 ExtraBold</family>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="title">
         <string>각인</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <layout class="QHBoxLayout" name="hLayoutNormalEngrave"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
